docs/get.md

```diff
@@ -0,0 +1,19 @@
+# Get
+
+The get command supports the following sub-commands:
+
+* [Recovery-profile](#recovery-profile)
```

Comment:
```suggestion
* [recovery-profile](#recovery-profile)
```

Reply:
this auto correct by vs code, probably due to `markdown` extension

Reply:
never-mind, I found the fix